constants/applicationStatus.js

```diff
@@ -0,0 +1,8 @@
+const ApplicationStatus = {
+  APPROVED: "Approved",
+  IN_REVIEW: "In Review",
+  RECEIVED: "Received",
+  UNAPPROVED: "Unapproved"
```

Comment:
What about `DENIED: "Denied"`, so `Unapproved` doesn't get misinterpreted as `In Review` ?

Reply:
Let's address this in subsequent PRs.

Reply:
Fair enough.